gulpfile.js

```diff
@@ -31,16 +35,21 @@ gulp.task('test:browser:before', (done) => {
     swarmB = new Swarm(peerB)
 
     swarmB.transport.add('ws', new WebSockets())
-    swarmB.transport.listen('ws', {}, null, cb)
+    swarmB.transport.listen('ws', {}, null, () => {
+      // swarmB.connection.addStreamMuxer(spdy)
+      // swarmB.connection.reuse()
```

Comment:
nop. It doesn't even happen with websockets just in Node.js, it only happens in the browser tests